libs/live-nft/src/__tests__/index.test.ts

```diff
@@ -66,4 +75,15 @@ describe("helpers", () => {
     const NFTs_TEST = accounts.map(a => a.nfts).flat() as ProtoNFT[];
     expect(orderByLastReceived(accounts, NFTs.concat(NFTs_TEST)).length).toBeGreaterThanOrEqual(1);
   });
+
+  it("mapChain", () => {
```

Comment:
a clearer test title is recommended in this test. 
Suggestion `it("should map a single chain")`